### PR TITLE
Toast alert feature

### PR DESCRIPTION
**Pull request by AshwinNS**
Adding toast alert feature.

### CODE REVIEW

File: sweetify/sweetify.py [hunk 1]
```diff
@@ -55,14 +65,26 @@ def _treat_data(opts):
...
     return opts
 
+
 def sweetalert(request, title, **kwargs):
     opts = DEFAULT_OPTS.copy()
     opts.update(kwargs)
     opts['title'] = title
-    opts = _treat_data(opts)    
+    if opts.pop('is_toast', None):
+        opts.pop('allowOutsideClick', None)
```
Comment (AshwinNS) [inline sweetify/sweetify.py:74]: `toast` does not support `allowOutsideClick` config.

**Comment by AshwinNS** [comment]
Any updates ?

**Comment by Atrox** [comment]
Sorry for my late response. I've been very busy lately. I'll try to find some time for this PR next week :)

**Comment by AshwinNS** [comment]
> Sorry for my late response. I've been very busy lately. I'll try to find some time for this PR next week :)

I'm waiting for this PR to merge to complete one deployment. :)

**Comment by Atrox** [comment]
I just published [v2.2.0](https://pypi.org/project/sweetify/2.2.0/). Thank you for the PR :)

**Pull request closed without merging**